api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -171,6 +171,12 @@ message Extraction {
   // If your regex contains capturing groups, use this field to determine which
   // group should be selected.
   uint32 subgroup = 3;
+
+  // The string to replace the matched portion of the source with
+  google.protobuf.StringValue replacement_text = 5;
+
+  // If set to true, all matches of the regex in the source will be replaced by the replacement_text.
```

Comment:
TODO: update commentary to indicate that this is mutually exclusive with `subgroup`

Reply:
I was worried about it breaking existing extractor configuration but I haven't actually tried it out -- let me take a look and I'll report back